modules/ROOT/pages/_partials/task-configure-proxy.adoc

```diff
@@ -0,0 +1,72 @@
+//tag::first-steps[]
+. In API Manager, click *Add API* and select *Add new API*.
+. Select *Mule Gateway* as your runtime.
+. Select *Deploy a proxy application* for *Proxy type*.
+//end::first-steps[]
+//tag::target-type[]
+. Select your *Target type* from the following options: CloudHub, Hybrid, or Runtime Fabric.
```

Comment:
Something to think about for the future - I don't think it's always necessary to list the options (as in line 7), because users will see this in the GUI. Doing this makes make doc maintenance harder because if an option is added or removed, you have to remember to fix the text. Also, it adds words that are unneeded in most instances.

Reply:
ok, good point